hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -1071,6 +1075,32 @@ public BoundaryStateChangeListener boundaryStateChangeListener() {
         return boundaryStateChangeListener;
     }
 
+    @Override
+    public Bytes encodeSystemTransaction(@NonNull StateSignatureTransaction stateSignatureTransaction) {
+        final var config = appContext.configSupplier().get().getConfigData(HederaConfig.class);
+        final var instant = appContext.instantSource().instant();
+        final var nodeAccountID = appContext.selfNodeInfoSupplier().get().accountId();
+
+        final var transactionValidStart =
```

Comment:
Afaict the `transactionValidDuration` and `transactionID#transactionValidStart` fields have only very minimal verification [here](https://github.com/hashgraph/hedera-services/blob/main/hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/TransactionChecker.java#L250) during `PreHandleWorkflow` and we could set them to,
 - `Duration.DEFAULT`
 - `Timestamp.DEFAULT`

respectively. Unless we want to change that, I would suggest using these `DEFAULT` values to (1) save a few bytes and (2) avoid implying to the reader taht these transactions get deeper validation downstream.

Reply:
Ah, good point. For some reason, I was convinced we would check if the time interval was valid. But we cannot do it at this point, of course. I will update the code. Thanks!